src/components/d3/BreakPointGroup.js

```diff
@@ -0,0 +1,43 @@
+const React = require('react');
+
+const BreakPointGroup = React.createClass({
+  propTypes: {
+    adjustedHeight: React.PropTypes.number.isRequired,
+    breakPointDate: React.PropTypes.number.isRequired,
+    breakPointLabel: React.PropTypes.string.isRequired,
+    margin: React.PropTypes.object.isRequired,
+    translation: React.PropTypes.string,
+    xScaleValueFunction: React.PropTypes.func.isRequired
+  },
+
+  getDefaultProps () {
+    return {
+      translation: 'translate(0,0)'
+    };
+  },
+
+  render () {
+    const breakPointLabelOffSet = 10;
+
+    return (
+      <g className='break-point-items' ref='breakPointItems' transform={this.props.translation}>
+        <line
+          className='break-point-line'
+          x1={this.props.xScaleValueFunction(this.props.breakPointDate)}
+          x2={this.props.xScaleValueFunction(this.props.breakPointDate)}
```

Comment:
Did you mean to call `{this.props.xScaleValueFunction(this.props.breakPointDate)}` twice here for x1 and x2? If so, you could call it outside the `render` function and assign it to a variable and use it here.